test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -151,3 +151,18 @@ export const passwordLockMetamaskExtension = async (
  */
 export const escapeColon = (selector: string): string =>
   selector.replace(':', '\\:');
+
+/**
+ * Wraps a describe call in a skip call if the SELENIUM_BROWSER environment variable is not chrome.
+ *
+ * @param description - The description of the test suite.
+ * @param callback - The callback function to execute the test suite.
+ */
+export const describeChromeOnly = (
+  description: string,
+  callback: () => void,
+) => {
+  return process.env.SELENIUM_BROWSER === 'chrome'
+    ? describe(description, callback)
+    : describe.skip(description, callback);
+};
```

Comment:
We're going to need a Firefox only version of this at some point. Would it make sense to make this helper accept a `browser: Browser` arg instead of making it chrome specific here? Then it could be used something like `describeOnlyBrowser(Browser.CHROME)` `describeOnlyBrowser(Browser.FIREFOX)`? 

Reply:
Ahh good point. Firefox will have a different entrypoint into the Multichain API, but that'll be handled at the test dapp layer, not the wallet.

Reply:
Its possible we'll use different tests for firefox? Probably not though we'd probably switch the transport in the test based on the environment... 

I'll just make this change for now since I'm in there fixing the lint issue and will need re-review anyways

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/30937/commits/101d29f31adb4020bd10c365877a9a2cb5e456ee